index.html

```diff
@@ -10,7 +10,68 @@
     <link rel="stylesheet" href="styles/index.css" />
 </head>
 <body>
-    <script src="./src/index.js"></script>
+
+<section>
+    <header>
+    <span id="text_weather">
+    <h1 > Weather Report</h1>
+        <span id="for_city"> <h3>For the city of</h3>
+            <div id="output">Seattle</div>
+        </span>
+        
+    </span>
+    </header>
```

Comment:
Indentation & spacing nits
```suggestion
    <header>
      <span id="text_weather">
        <h1> Weather Report</h1>
        <span id="for_city">
          <h3>For the city of</h3>
          <div id="output">Seattle</div>
        </span>    
      </span>
    </header>
```